docs/src/routes/docs/[...3]modules/core.md

```diff
@@ -186,11 +204,13 @@ type i18nOptions = Record<Locale, i18n>
 To see a list of all of the text values that can be internationalized or replaced, check out the [default en file](https://github.com/blocknative/web3-onboard/blob/develop/packages/core/src/i18n/en.json).
 Onboard is using the [ICU syntax](https://formatjs.io/docs/core-concepts/icu-syntax/) for formatting under the hood.
 
-**`theme`**
+#### **theme**
+
 A string or an object that defines the color theme web3-onboard will render the components.
-Define a custom or predefined theme for Web3Onboard using either: 
-  * BuiltInThemes: ['default', 'dark', 'light', 'system']
-  * ThemingMap object to create a totally custom theme
+Define a custom or predefined theme for Web3Onboard using either:
+
+- BuiltInThemes: ['default', 'dark', 'light', 'system']
```

Comment:
@Adamj1232 is there a reason this is in TitleCase? if it is a variable name from code then maybe it would make sense to wrap in a code block? `BuiltInThemes`